lib/engine/game/g_1856.rb

```diff
@@ -309,6 +311,33 @@ def cert_limit
         POST_NATIONALIZATION_CERT_LIMIT[num_corporations][@players.size]
       end
 
+      def destination_check
+        @corporations.each do |corp|
+          # The capitalization method of unparred corporations is nil
+          next unless corp.capitalization == :escrow
+
+          destinated!(corp) if hexes_connected?(@destinations[corp.id].first, @destinations[corp.id].last)
```

Comment:
goal_hex_ids is one hex or an array?

Reply:
seems like it should just be start_hex and end_hex, but you have goal_hex_ids.include?(...)

Reply:
It's an array to handle the 2-hex offboard destination (for CV iirc) in the alternate destinations for 1856